main.py

```diff
@@ -14,6 +15,22 @@
 from .settings import CONSISTENCY_INTERVAL, FLOWS_DICT_MAX_SIZE
 
 
+def casting_fields(flow_dict):
+    """Make casting the match fields from UBInt() to native int ."""
+    match = flow_dict['match']
+    for field, value in match.items():
+        if isinstance(value, UBInt8):
```

Comment:
[UBIntBase](https://github.com/kytos/python-openflow/blob/27fa621500a538d991172abe2f5a8a6b71446a9d/pyof/foundation/base.py#L262) was created to avoid this repetition.

Reply:
Resolved in https://github.com/kytos/flow_manager/pull/127/commits/20fb784e7d82ce15e6b9f6cab5408d397349305d